test/build-images.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e 	# Exit immediately upon failure
+
+: ${1?"Need to pass the CLI channel for which to build the Docker images"}
+
+repo_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
+docker_repo="microsoft/dotnet-nightly"
+cli_channel="${1}"
+
+if [ ! -d "${repo_root}/${cli_channel}" ]; then
+    echo "Error: The CLI channel '${cli_channel}' does not exist in this repo."
+    exit 1
+fi
+
+tag_base="${docker_repo}:${cli_channel}"
+image_dir_base="${repo_root}/${cli_channel}/debian"
+
+echo "----- Building ${tag_base}-sdk -----"
+docker build -t ${tag_base}-sdk ${image_dir_base}
+
+echo "----- Building ${tag_base}-core -----"
+docker build -t ${tag_base}-core ${image_dir_base}/core
+
+echo "----- Building ${tag_base}-core-deps -----"
+docker build -t ${tag_base}-core-deps ${image_dir_base}/core-deps
+
+echo "----- Building ${tag_base}-onbuild -----"
+docker build -t ${tag_base}-onbuild ${image_dir_base}/onbuild
```

Comment:
I was thinking that the script would self discover the dockerfiles, build them, and then infer the tag name from the directory structure.  This will be less maintenance overtime as tags are added/removed.